trackma/ui/qt/models.py

```diff
@@ -360,7 +353,7 @@ def data(self, index, role):
             return self.thumbs.get(row)
         elif role == QtCore.Qt.BackgroundRole:
             t = self.results[row].get('type')
-            if t == utils.Type.TV:
+            if t == utils.Type.TV or t == utils.Type.ONA:
```

Comment:
As I mentioned in #781 I feel like ONAs are more similar than OVAs than TV shows - could we use the OVA color instead? Or, maybe a new one?